.changeset/chilly-windows-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Not sure why two changeset files were created. Maybe because the PR was renamed? But you can probably just delete both of them to reset the action

Reply:
Oh, this is a mistake. Will delete one.